partner_firstname/models/partner.py

```diff
@@ -145,3 +188,10 @@ def _install_partner_firstname(self):
         # Force calculations there
         records._inverse_name()
         _logger.info("%d partners updated installing module.", len(records))
+
+    # Python constraint gives a more explicit error
+    _sql_constraints = [(
+        'check_name',
+        "CHECK( 1=1 )",
```

Comment:
Ah OK. As you can see this code at first sight seems strange. Could you please put a more descriptive comment above? Something like `Disabling SQL constraint from base module because Python constraint gives a more explicit error`.